$NetBSD: patch-.._scripts_install__fcitx5__bazel,v 1.1 2024/02/10 12:26:01 ryoon Exp $

Install below DESTDIR, and leave the data and icons to the package.

The script writes straight into PREFIX, which during a pkgsrc build has to
be the staging directory instead.  install -D is a GNU extension the BSDs
do not have; the directories are created by the package before this runs.
The two scripts it chains to are called from the Makefile, where their
output can be listed in the PLIST.

--- ../scripts/install_fcitx5_bazel.orig	2023-10-26 15:09:25.000000000 +0000
+++ ../scripts/install_fcitx5_bazel
@@ -1,7 +1,7 @@
 #!/bin/sh
-PREFIX="${PREFIX:-/usr}"
+PREFIX="${DESTDIR}${PREFIX:-/usr}"
 
-install -D -m 755 "bazel-bin/unix/fcitx5/fcitx5-mozc.so" "${PREFIX}/lib/fcitx5/fcitx5-mozc.so"
+install -m 755 "bazel-bin/unix/fcitx5/fcitx5-mozc.so" "${PREFIX}/lib/fcitx5/fcitx5-mozc.so"
 
-PREFIX="${PREFIX}" ../scripts/install_fcitx5_data
-PREFIX="${PREFIX}" ../scripts/install_fcitx5_icons
+#PREFIX="${PREFIX}" ../scripts/install_fcitx5_data
+#PREFIX="${PREFIX}" ../scripts/install_fcitx5_icons
